src/MultiThreadedDemo.java

```diff
@@ -1,24 +1,39 @@
+import sun.security.util.BigInt;
+
+import java.math.BigInteger;
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Executors;
 
 public class MultiThreadedDemo {
 	//Initialize constants
 	public static final int MAXTHREADS = 7;
 	public static final long TOTALPRIMENUMS = 10000;
+	public static final BigInteger MAXNUMTOCHECK = BigInteger.valueOf(Long.MAX_VALUE + 1);
 	public static final int SLEEPTIME = 500;
+	public static final boolean USEBIGINT = false;
```

Comment:
Its meant to be edited before running the program similarly to how you need to edit the other variables. My intention was to wrap around the existing program and leave the ability to still run the program using `long` if one wanted to.